src/ipinfo.rs

```diff
@@ -72,13 +74,13 @@ impl Default for IpInfoConfig {
 pub struct IpInfo {
     url: String,
     token: Option<String>,
-    client: reqwest::Client,
+    client: reqwest::blocking::Client,
```

Comment:
@fayzanx I am assuming we can use `reqwest::blocking::Client` for now as there are no async operations but we need to go back to the async client for the batch operations iterator that we planned to implement later on.